hack/devtools/platform_msi.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# This script creates a platform Identities to use for local development
+# The script reads the operators file and creates platform identities for each operator
+
+prompt_for_operators_file() {
```

Comment:
Should this be 
```suggestion
prompt_for_platform_workload_identity_role_set() {
```
instead, or what is the operators file in this case, just a list of operators only?

It might require a little jq magic, but I'm thinking we could use a platformworkloadidentityroleset instead since we already have that defined (see description of https://github.com/Azure/ARO-RP/pull/3617)

What do others think?

Reply:
@cadenmarchese  So, if I understand correctly, what you're suggesting is, to ask the user to set env var PLATFORM_WORKLOAD_IDENTITY_ROLE_SETS and read the values from this env var instead of file?

Reply:
@gouthamMN I'd be open to that idea or Kipp's idea, whichever we prefer. The nice thing about the PLATFORM_WORKLOAD_IDENTITY_ROLE_SETS is that it also contains the role assignments that need to be done, so if you took it in as an environment variable you could extend the functionality of the script to also assign the roles.